src/components/checkDraw.js

```diff
@@ -0,0 +1,9 @@
+export function checkDraw(gameState, winningMessage) {
+  if (
```

Comment:
Add a comment with an explanation of what happens here

Reply:
added:
// checking if all of the board position are changed without the winner